### PR TITLE
updated travisci readme document and fixes issue #2248

### PR DESCRIPTION
**Pull request by rameshthoomu**
## Description

Reference has been created for Travis CI readme in docs/index.md document also changed the travis settings images reference path. This PR also fixes issue #2248. 
## Motivation and Context

Fixes #
#2248
## How Has This Been Tested?

Only documentations changes has been updated. 
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by:thoomu@us.ibm.com

### CODE REVIEW

**Comment by rameshthoomu** [comment]
@christo4ferris : Could you please review this?

**Comment by gabre** [comment]
@rameshthoomu I had a look at the text and the new pic. Seems OK to me and I think it could be merged. @christo4ferris

File: docs/index.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Below, you'll find the following sections:
   - [Chaincode developer guide](#chaincode-developer-guide)
   - [API developer guide](#api-developer-guide)
 - [Operations guide](#operations-guide)
+- [Continuous Integration](https://github.com/hyperledger/fabric/blob/master/TravisCI_Readme.md)
```
Comment (christo4ferris) [inline docs/index.md]: @rameshthoomu Why are we bringing this back into the docs? Travis will soon be EOL'ed and it is really irrelevant to anyone but the CI team. I explicitly removed it from the index previously because I did not think it belonged in the end user/developer docs, and at the end of the day, that is what this is really supposed to be.

Comment (rameshthoomu) [inline docs/index.md]: @christo4ferris : I thought this was missed in previous fixes. Now, I completely agree with you.  Will revert the changes related to Travis doc.

Comment (christo4ferris) [inline docs/index.md]: cool thanks... it is great that we have this, but not part of user docs.

**Comment by srderson** [comment]
LGTM
